lib/api.common.js

```diff
@@ -594,18 +594,17 @@ api.common.sortCompareByName = (
   return 0;
 };
 
-api.common.clearCacheStartingWith = (
+api.common.clearCacheStartsWith = (
```

Comment:
Maybe it is even more appropriate to indicate somehow that _names_ are starting with something.